java/common/common-task/src/main/java/sleeper/common/task/RunCompactionTasks.java

```diff
@@ -15,236 +15,37 @@
-                .cluster(instanceProperties.get(COMPACTION_CLUSTER))
-                .overrides(override)
-                .propagateTags(PropagateTags.TASK_DEFINITION);
-
-        CompactionECSLaunchType launchType = instanceProperties.getEnumValue(COMPACTION_ECS_LAUNCHTYPE, CompactionECSLaunchType.class);
-        if (launchType == CompactionECSLaunchType.FARGATE) {
-            String fargateVersion = Objects.requireNonNull(instanceProperties.get(FARGATE_VERSION), "fargateVersion cannot be null");
-            NetworkConfiguration networkConfiguration = networkConfig(instanceProperties);
-            runTaskRequest
-                    .launchType(LaunchType.FARGATE)
-                    .platformVersion(fargateVersion)
-                    .networkConfiguration(networkConfiguration)
-                    .taskDefinition(instanceProperties.get(COMPACTION_TASK_FARGATE_DEFINITION_FAMILY));
-        } else if (launchType == CompactionECSLaunchType.EC2) {
-            runTaskRequest
-                    .launchType(LaunchType.EC2)
-                    .taskDefinition(instanceProperties.get(COMPACTION_TASK_EC2_DEFINITION_FAMILY));
-        } else {
-            throw new IllegalArgumentException("Unrecognised ECS launch type: " + launchType);
-        }
-        return runTaskRequest.build();
-    }
-
-    /**
-     * Create the container definition overrides for the task launch.
-     *
-     * @param  instanceProperties the instance properties
-     * @return                    the container definition overrides
-     */
-    private static TaskOverride createOverride(InstanceProperties instanceProperties) {
-        ContainerOverride containerOverride = ContainerOverride.builder()
-                .name(COMPACTION_CONTAINER_NAME)
-                .command(List.of(instanceProperties.get(CONFIG_BUCKET)))
-                .build();
-        return TaskOverride.builder()
-                .containerOverrides(containerOverride)
-                .build();
-    }
-
-    /**
-     * Create the container networking configuration.
-     *
-     * @param  instanceProperties the instance properties
-     * @return                    task network configuration
-     */
-    private static NetworkConfiguration networkConfig(InstanceProperties instanceProperties) {
-        AwsVpcConfiguration vpcConfiguration = AwsVpcConfiguration.builder()
-                .subnets(instanceProperties.getList(SUBNETS))
-                .securityGroups(instanceProperties.getList(ECS_SECURITY_GROUPS))
-                .build();
-        return NetworkConfiguration.builder()
-                .awsvpcConfiguration(vpcConfiguration)
-                .build();
-    }
 
     public static void main(String[] args) {
         if (args.length != 2) {
             System.out.println("Usage: <instance-id> <number-of-tasks>");
             return;
         }
-        String instanceId = args[0];
-        int numberOfTasks = Integer.parseInt(args[1]);
```

Comment:
I think it's a bit easier to read with the arguments brought out like this, since you can compare it against the usage and count check just above here more directly. Can we keep this as it was before? The same applies to BulkExportTaskHostScaler.